clusterloader2/testing/load/config.yaml

```diff
@@ -849,63 +849,7 @@ steps:
       bigServicesPerNamespace: 0
 {{end}} # not EXIT_AFTER_EXEC
 
-- name: Collecting measurements
```

Comment:
This is great, but let's also have the module to cover starting the measurements. So lines 129-185. There will be some diff reported by my tool, as we're not setting the same params when creating and when gathering, but this diff is expected and acceptable.

Reply:
covered, thank you @mm4tt!